spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/StructAccessor.java

```diff
@@ -91,6 +94,7 @@
 				.put(double[].class, AbstractStructReader::getDoubleArray)
 				.put(long[].class, AbstractStructReader::getLongArray)
 				.put(boolean[].class, AbstractStructReader::getBooleanArray)
+				.put(BigDecimal.class, AbstractStructReader::getBigDecimal)
```

Comment:
This looks like a duplicate of the above map, but there's probably some obscure reason for it. :)